api/controllers/shabads.js

```diff
@@ -24,7 +27,15 @@ const allColumns = `v.ID, v.Gurmukhi, v.GurmukhiUni, v.Translations, v.PageNo AS
 const allColumnsWhere = 'AND s.ShabadID < 5000000';
 
 const error = (err, res) => {
-  res.status(400).json({ error: true, data: err });
+  console.error(err);
+  Error.captureStackTrace(err);
+  res.status(400).json({
+    error: true,
+    data: {
+      error: JSON.stringify(err),
+      stack: JSON.stringify(err.stack),
```

Comment:
why `stringify` here? `res.json` does that automatically. [Docs](https://expressjs.com/en/api.html#res.json)